src/ore/Cargo.toml

```diff
@@ -22,6 +22,7 @@ clap = { version = "3.2.20", features = ["env"], optional = true }
 ctor = { version = "0.1.23", optional = true }
 either = "1.8.0"
 futures = { version = "0.3.24", optional = true }
+mz-build-info = { path = "../build-info" }
```

Comment:
~Aw, hm, a bit of a bummer to make ore dependent on `mz-build-info`, as I try to keep `mz-ore` relatively free of Materialize-specific stuff, and `mz-build-info` has some Materialize-specific stuff.~ Seems worth doing, though; anything else would be real painful.

Reply:
oh I think we can work around this without too much pain. we can just pass in a string or two for the build info